src/server/server.test.js

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest';
+import * as after from '@jaredpalmer/after';
+import server from './index';
+
+describe('Server', () => {
+  const makeRequest = async path => request(server).get(path);
+
+  it(`should not pass an 'x-powered-by' response header`, async () => {
+    const { headers } = await makeRequest('/status');
+    const headerKeys = Object.keys(headers);
+    expect(headerKeys).not.toContain('x-powered-by');
+  });
+
+  describe('/status', () => {
+    it('should respond with a 200', async () => {
+      const { statusCode } = await makeRequest('/status');
+      expect(statusCode).toBe(200);
+    });
+  });
+
+  describe('/*', () => {
+    const testResponseText = (testTitle, value, mockRender, testAssertion) => {
+      it(testTitle, async () => {
+        after.render = jest.fn().mockImplementationOnce(mockRender);
```

Comment:
Is there a reason why you're using After's render function here?

Reply:
This mocks the render function that is called in the `/*` so that we can make assertions about it.

Reply:
Ah, yes, of course. :+1: Makes sense.